src/commands/hearthstone.js

```diff
@@ -0,0 +1,38 @@
+const Aquarius = require('../aquarius');
+const hdb = require('hearthstone-db');
+
+class Hearthstone extends Aquarius.Command {
+  constructor() {
+    super();
+    this.description = 'Displays Hearthstone Card Image';
+  }
+
+  helpMessage() {
+    let msg = super.helpMessage();
+
+    msg += 'Usage:\n';
+    msg += '```[[Tirion Fordring]]```';
+    return msg;
+  }
+
+  message(msg) {
+    const hearthInput = Aquarius.Triggers.cardTrigger(msg);
+
+    if (hearthInput) {
+      hearthInput.forEach(cardInput => {
+        let cardName = `${cardInput[1]}`;
+        this.log(`Request for ${cardName}`);
+
+        Aquarius.Loading.startLoading(msg.channel);
```

Comment:
Since we're not hitting an API, I'm not sure we need to show a loading indicator... how long does the bot take to respond? Not sure how long a search would take since I'm not familiar with how many Hearthstone cards there are.